io.github.narunlifescience.AlphaPlot.yml

```
@@ -0,0 +1,78 @@
+app-id: io.github.narunlifescience.AlphaPlot
+runtime: org.kde.Platform
+runtime-version: "5.15"
+sdk: org.kde.Sdk
+command: alphaplot
+rename-icon: alphaplot
+rename-desktop-file: alphaplot.desktop
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --device=dri
+  - --env=SESSION_MANAGER= # required to avoid Qt warning
+  - --filesystem=xdg-desktop
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-public-share
+cleanup:
+  - /include
+  - /lib/mkspecs
+  - /lib/pkgconfig
+  - /lib/*/cmake
+  - /lib/*/pkgconfig
+  - /share/aclocal
+  - /share/info
+  - /share/man
+  - "*.a"
+  - "*.la"
+  - "*.prl"
+modules:
+  - name: gsl
+    config-opts:
+      - --disable-static
+    cleanup:
+      - /bin
+    sources:
+      - type: archive
+        url: https://ftpmirror.gnu.org/gnu/gsl/gsl-2.6.tar.gz
+        sha256: b782339fc7a38fe17689cb39966c4d821236c28018b6593ddb6fd59ee40786a8
+
+  - shared-modules/glu/glu-9.json
+
+  - name: qt5-qtdatavis3d
+    buildsystem: qmake
+    cleanup:
+      - /bin
+    make-install-args:
+      - INSTALL_ROOT=/app/scratchdir
+    sources:
+      - type: archive
+        url: http://download.qt.io/archive/qt/5.15/5.15.1/submodules/qtdatavis3d-everywhere-src-5.15.1.tar.xz
+        sha256: 89ed596ea452a8dd8223d094690606bcccc92962776584aceefcc13f56538c06
+    post-install:
+      - cp -r /app/scratchdir/usr/* /app
+      - rm -rf /app/scratchdir
+      - ln -s /app/lib/*/*.so* /app/lib
+      - sed -e 's@$$QT_MODULE_LIB_BASE@/app/lib@' -i /app/lib/mkspecs/modules/*.pri
+
+  - name: alphaplot
+    buildsystem: qmake
+    build-options:
+      env:
+        - QMAKEPATH=/app/lib
+    config-opts:
+      - QMAKE_INCDIR+=/app/include/QtDataVisualization
+    sources:
+      - type: git
+        url: https://github.com/narunlifescience/AlphaPlot.git
+        tag: 1.011
```

```suggestion
        tag: "1.011"
```
Nitpick: won't trust yaml that much.